engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -222,28 +227,21 @@ private void standPlayer(EntityRef entity) {
      */
     private void stopAutoMove() {
         isAutoMove = false;
+        List<Input> inputs = bindsConfig.getBinds(new SimpleUri("engine:forwards"));
+        Input forwardKey = inputs.get(0);
```

Comment:
yeah, that's exactly what I was thinking about (the question was a bit unclear, though). Since you cannot assume that the player has defined that key, you should check for the empty list and do nothing (instead of provoking the NPE). 
It _should_ not happen, but a hard crash is not what we want here.